lesson2/homework/helloworld/src/main/java/Animal.java

```diff
@@ -20,7 +20,8 @@ public Animal(int n_legs, String n_name, String n_color, String n_species, doubl
         species = n_species;
         weight = n_weight;
     }
-//Got dem setters
+
+    //Got dem setters
```

Comment:
Make your comments the same indent-level as the code it's referring to, and it's a good idea to add an empty line in between chunks of code to make it more readable at a glance